custom/conf/app.example.ini

```diff
@@ -1187,6 +1187,14 @@ QUEUE_CONN_STR = "addrs=127.0.0.1:6379 db=0"
 MAX_ATTEMPTS = 3
 ; Backoff time per http/https request retry (seconds)
 RETRY_BACKOFF = 3
+; Allowlist for migrating, default is blank. Blank means everything will be allowed.
+; Multiple domains could be separated by commas.
+ALLOWLISTED_DOMAINS =
+; Blocklist for migrating, default is blank. Multiple domains could be separated by commas.
+; When ALLOWLISTED_DOMAINS is not blank, this option will be ignored.
+BLOCKLISTED_DOMAINS =
+; Allow private addresses defined by RFC 1918, RFC 1122, RFC 4632 and RFC 4291 (false by default)
+ALLOW_LOCALNETWORKS = false
```

Comment:
Hmm, these pref names seem a bit too generally named, how about `MIGRATE_ALLOWED_DOMAINS`, `MIGRATE_BLOCK_DOMAINS` and `MIGRATE_ALLOWED_NETWORKS` (defaulting to a list of CIDRs), possibly also `MIGRATE_BLOCK_NETWORKS`.

Reply:
@silverwind they are already in the `[migrations]` section so this should be fine ...

Reply:
hmm, I'd at least rename to `{BLOCKED,ALLOWED}_{DOMAINS,NETWORKS}`. I do think allowing to specify CIDRs would be quite nice to have too.